processor/resourcedetectionprocessor/README.md

```diff
@@ -40,6 +40,19 @@ to read resource information from the [GCE metadata server](https://cloud.google
     * host.name
     * host.type
 
+It also can optionally gather tags for the EC2 instance that the collector is running on
+
+EC2 custom configuration example:
```

Comment:
From some searching, it seems that hosts by default don't have permissions to call `describeTags` API, which makes sense since it's independent from IMDS. Is there any documentation we can add for special setup required by this processor setting?

Reply:
I have added in a note with requirements for permissions, let me know if it is clear

Reply:
That looks good to me! 👍